automod/rules/interaction.go

```diff
@@ -25,6 +25,7 @@ func InteractionChurnRule(c *automod.RecordContext) error {
 			c.Logger.Info("high-like-churn", "created-today", created, "deleted-today", deleted)
 			c.AddAccountFlag("high-like-churn")
 			c.ReportAccount(automod.ReportReasonSpam, fmt.Sprintf("interaction churn: %d likes, %d unlikes today (so far)", created, deleted))
+			c.EscalateAccount()
```

Comment:
I'd be in favor of commenting this out for now, so we can review and get the engine change merged, then do the policy/behavior change separately.